### PR TITLE
[BOS-2019] Sponsor update - week of 06/02.

### PR DESCRIPTION
**Pull request by don-code**
Adding sponsors identified at the weekly sync on 06/02.

Screenshot from 2019-06-03 23-30-14

### CODE REVIEW

**Comment by bridgetkromhout** [comment]
Hi - that same person from Google has been sponsoring most events as "Google Cloud" with the GCP logo. Are you certain they want the generic "Google"? Thanks.

**Comment by bridgetkromhout** [comment]
@nathenharvey for checking on the Google/GCP logo/link.

File: data/events/2019-boston.yml [hunk 1]
```diff
@@ -109,6 +109,14 @@ sponsors:
     level: gold
   - id: arresteddevops
     level: community
+  - id: google
```
Comment (nathenharvey) [inline data/events/2019-boston.yml]: Please use `googlecloud` here.

Thanks

**Comment by nathenharvey** [comment]
Thanks for updating that, @bridgetkromhout.  LGTM

**Pull request merged**